.github/actions/stripe_setup/action.yml

```diff
@@ -0,0 +1,12 @@
+name: "Stripe Setup"
+description: "Sets up the GitHub Actions Workflow"
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-java@v3
+      with:
+        distribution: 'corretto'
+        java-version: '11'
+    - name: Setup Gradle
```

Comment:
Re my `gradle.properties` comment - we could set it up here so it applies to all actions.

Reply:
Good idea. Thanks!